lib/util/clone.js

```diff
@@ -0,0 +1,29 @@
+// Simple clone function, which creates a deep copy of the given node
+// And recurses on the children (due to the shallow nature of the mathjs node
+// clone)
+function clone(node) {
+  const copy = node.clone()
+
+  if (node.changeGroup) {
+    copy.changeGroup = node.changeGroup
+  }
+
+  if (node.args) {
+    node.args.forEach((child, i) => {
+      copy.args[i] = clone(child)
+    })
+  } else if (node.content) {
+    copy.content = clone(node.content)
+  }
+
+  if (node._kemuIsSorted) {
```

Comment:
Some names left after "kemu studio" extended code... But after discussion we decided to clean up later with bigger batches.